packages/wallet/src/components/settings/PersonalSettingsForm.tsx

```diff
@@ -99,6 +113,23 @@ export const PersonalSettingsForm = ({
           </Button>
         </div>
       )}
+      <div className="mt-6 mb-4">
```

Comment:
Address and email will just be displayed, disabled inputs, moved edit button bellow the editable fields.